src/app/checklists/items-list/item/item.component.scss

```diff
@@ -132,19 +134,24 @@
   width: 300px;
 }
 
-.prompt-tabulator:first-child,
-.prompt:first-child:not(.item-centered > span) {
-  padding-left: 10px;
-  padding-right: 10px;
-}
-
 .prompt-tabulator {
   opacity: 33%;
-  width: 25px;
-  margin-top: -3px;
+  width: 26px;
+  margin-top: 3px;
 }
 
 .item-container:hover .prompt-tabulator:before,
 .item-container:focus .prompt-tabulator:before {
   content: '⟷';
 }
+
+/* Full with editors for non-centered, non-challenge-response items */
+
+.item:not(.item-challenge-response):not(.item-centered) .prompt {
+  display: flex;
```

Comment:
I like this! Thanks again for the contribution!

Why not for centered though? I tried it and it seems to work ok there:

Before:
<img width="1470" height="96" alt="image" src="https://github.com/user-attachments/assets/160ca5f0-8fde-4d1c-8511-0b7133c3079b" />

After:
<img width="1322" height="118" alt="Screenshot 2025-09-22 at 01 16 01" src="https://github.com/user-attachments/assets/0a933072-3cbe-4cfd-9012-867d2c5b6b51" />

After when not editing:
<img width="1323" height="115" alt="Screenshot 2025-09-22 at 01 17 19" src="https://github.com/user-attachments/assets/84bbc28c-2df3-46ed-a043-87cd45743265" />

Reply:
The reason I excluded it is due to other overhanging items that can be centered, like WARNING/CAUTION/NOTES. The width cannot be 100% for them. We should add a `.editing` annotation to `.item` level and format these separately so we can address them during editing. We can do that later and I thought best to keep it short and not broken for now.